apps/mosaico/static/fragor-och-svar.js

```diff
@@ -0,0 +1,15 @@
+var questions = document.getElementsByClassName("faq__question");
+var i;
+for (i = 0; i < questions.length; i++) {
```

Comment:
This could have been written as just one event handler on `.static-page__faq`. Events bubble to the containing element's event handler and it could catch any clicks to `.faq_question` elements.

Reply:
Yes, I shall look into that solution, but maybe for now it could be like this?